cmd/version/version.go

```diff
@@ -78,8 +82,65 @@ func NewCmdVersion(rootCmd *cobra.Command) *cobra.Command {
 			}
 
 			util.TablePrinter(util.VersionColumnDefinition, rows, printers.PrintOptions{Wide: true})
+			checkForLatestVersion(rootCmd.Version)
 		},
 	}
 	cmd.SetUsageTemplate(versionCmdHelp)
 	return cmd
 }
+
+func checkForLatestVersion(currVersion string) {
+	// getting the latest version of openebsctl from sigs.k8s.io/krew-index
+	resp, err := http.Get("https://raw.githubusercontent.com/kubernetes-sigs/krew-index/master/plugins/openebs.yaml")
+	if err != nil {
+		// The seperator for the error print
+		fmt.Println()
+		fmt.Fprintf(os.Stderr, "Error fetching latest version %s", err.Error())
+		return
+	}
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		// The seperator for the error print
+		fmt.Println()
+		fmt.Fprintf(os.Stderr, "Error reading response body %s", err.Error())
+		return
+	}
+
+	var data map[string]interface{}
+	err = yaml.Unmarshal(body, &data)
+	if err != nil {
+		// The seperator for the error print
+		fmt.Println()
+		fmt.Fprintf(os.Stderr, "Error parsing yaml %s", err.Error())
+		return
+	}
+
+	latestVersion := data["spec"].(map[string]interface{})["version"].(string)
+	if !isLatestVersion(currVersion, latestVersion) {
+		// The seperator for the error print
+		fmt.Println()
+		fmt.Printf("You are using an older version (%s) of cli, latest available version is: %s", currVersion, latestVersion)
+	}
+}
+
+func isLatestVersion(currVersion string, latestVersion string) bool {
+	if currVersion[0] == 'v' {
+		currVersion = currVersion[1:]
+	}
+
+	if latestVersion[0] == 'v' {
+		latestVersion = latestVersion[1:]
+	}
+
+	if currVersion[0] < '0' || currVersion[0] > '9' {
+		return true
+	}
+
+	if currVersion != latestVersion {
+		return false
+	}
+
+	return true
```

Comment:
```suggestion
	return latestVersion == currVersion
```

Go has a niche tool called gosimple or something similar name(I cannot recall it right away), this would likely suggest something similar. Lower LoC gets us lesser debugging time :)

Reply:
Actually that was the defensive code:
```
       if currVersion[0] == 'v' {
		currVersion = currVersion[1:]
	}
	if latestVersion[0] == 'v' {
		latestVersion = latestVersion[1:]
	}
```

to check if any of the two versions coming up with v appended in front then they could be checked properly.
But since We are the setter of the version, Are we pretty sure that we will be naming both versions (i.e. current one and latest one with v in front like v0.3.1)?
@vharsh

Reply:
Yes, the currentVersion can have arbitrary things like `dev` or a short commit-hash, but the latestVersion will ideally always begin with `v`.

Note: Updating the plugin version been automated by #87, it won't work the first time without human checks this month(15th September), but later on, it can be expected to work always.

Reply:
Done